### DISCUSSION TITLE
Welcome to scripts-blog Discussions!

### DISCUSSION THREAD

**Discussion by BanterBoy** [BanterBoy/scripts-blog#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by superswan** [comment]
This is pretty awesome. Came across your blog today while trying to find a nice script to setup the Central Store on a DC. I'll definitely be keeping a close eye on this. Thanks again.

**Reply by BanterBoy** [reply]
You are very welcome.

I have some more scripts and functions to add, hopefully I will have time soon 👍
